src/commands/deploy/deploy.ts

```diff
@@ -88,6 +95,48 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
     await notifyDeployComplete(context, node, context.workspaceFolder.uri.fsPath);
 }
 
+async function tryToUpdateWorkerProcessTo64BitIfRequired(context: IDeployContext, siteConfig: WebSiteManagementModels.SiteConfigResource, node: SlotTreeItemBase, language: ProjectLanguage): Promise<void> {
+    try {
+        const functionProject: string | undefined = await tryGetFunctionProjectRoot(context.workspaceFolder.uri.fsPath);
+        if (functionProject === undefined) {
+            throw new Error(localize('failedToFindFuncHost', 'Unable to locate Azure Functions project.'));
+        }
+        const projectFiles: string[] = await dotnetUtils.getProjFiles(language, functionProject);
+        if (projectFiles.length === 0) {
+            throw new Error(localize('unableToFindProj', 'Unable to detect project file.'));
+        }
+        const mainProject: string = path.join(functionProject, projectFiles[0]);
+        const platformTarget: string | undefined = await dotnetUtils.tryGetPlatformTarget(mainProject);
+        if (platformTarget === 'x64' && siteConfig.use32BitWorkerProcess === true) {
+            const message: string = localize('overwriteSetting', 'Detected 64 bit project. Update setting in Azure Portal?');
+            const deployAnyway: vscode.MessageItem = { title: localize('deployAnyway', 'Deploy Anyway'), isCloseAffordance: true };
+            const dialogResult: vscode.MessageItem = await ext.ui.showWarningMessage(message, { modal: true }, DialogResponses.yes, deployAnyway);
+            if (dialogResult.isCloseAffordance === true) {
```

Comment:
No need to do this `isCloseAffordance` stuff. Just check `if (dialogResult === deployAnyway) {`